pacai/bin/crawler.py

```diff
@@ -397,5 +397,6 @@ def _load_args(args):
     return max_steps
 
 if __name__ == '__main__':
+    logging.basicConfig(format='%(levelname)s - %(asctime)s - %(message)s', level=logging.INFO)
```

Comment:
Move all of these calls to a method in a centralized file, how about putting it in `util/logs.py`.
Maybe call the method `initLogging()`?

Note that not calling it `logging` is very purposeful.
We don't want it getting confused with the standard library, `logging`.